assets/src/components/ColorPaletteControl/ColorPaletteControl.js

```diff
@@ -7,13 +7,15 @@ import { ColorPalette } from '@wordpress/components';
 function ColorPaletteControl( { label, className, value, help, instanceId, onChange, options = [], ...passThroughProps } ) {
   const id = `inspector-color-palette-control-${ instanceId }`;
 
+  const optionsAsColors = options.map( option => ( { color: option.value } ) );
```

Comment:
The colors passed to `ColorPalette` can have a label. I hadn't added that to the config files, as I didn't have the names for the colors we use. We may want to use these labels at some point as they improve Accessibility, so maybe you can already pass on this label from the theme here?

Reply:
So I did this change, but now I'm starting to regret it, as we introduced another thing to maintain (checking that color names match the actual colors, which is quite impossible to test automatically) and this whole UI is far from accessible... also is quite weird to imagine a visually impaired person choosing color palettes (although it's possible). Also, I found that none of the colors in the campaign templates are present in `_colors.scss`, not even the GP greens 🤦 , so I'm making a script to actually collect the colors we are actually using in the whole site. Anyway, it's done, but let's try to keep the scope limited to what the tickets need.

Reply:
The accessibility concerns are more broad in the sense that any user can benefit from it: likely we will have campaign themes with slight variations of the same color. Including a textual name will make it a lot easier to communicate about them, compared to having only the visual appearance to go by.

Maybe we could also set the text to the hex code if none is present.

Reply:
I think we are overthinking this a bit, there is no standard for color names, so we'll have to start naming things as "Navy Blue", or "Lime Green". If we don't pass a name, the swatch actually gets the Hex value in the tooltip, so maybe we should just remove the labels, or I could add the Hex color to the label, as in: `Red (#FF0000)`. 🤔

Reply:
Keep in mind also, that there is some design work pending here too, to vet our full color palette and keep the ones that we actually want to use. So I wouldn't spend much time trying to figure out names for colors that maybe shouldn't even be used.

Reply:
Indeed we should not be setting up the names right now, I was just suggesting to pass on the label already so that adding the labels at a later point becomes an easy configuration only task.

Yeah also noticed that it already shows the hex code. We can re-iterate later if we want it to display differently.